react/features/participants-pane/components/breakout-rooms/components/native/CollapsibleRoom.js

```diff
@@ -39,42 +39,33 @@ function _keyExtractor(item: Object) {
 
 export const CollapsibleRoom = ({ room, searchString }: Props) => {
     const dispatch = useDispatch();
-    const [ collapsed, setCollapsed ] = useState(false);
     const { t } = useTranslation();
-    const _toggleCollapsed = useCallback(() => {
-        setCollapsed(!collapsed);
-    }, [ collapsed ]);
     const _openContextMenu = useCallback(() => {
         dispatch(openDialog(BreakoutRoomContextMenu, { room }));
     }, [ room ]);
+    const roomParticipantsNr = Object.values(room.participants || {}).length;
+    const title
+        = `${room.name
+    || t('breakoutRooms.mainRoom')} (${roomParticipantsNr})`;
+    const containerStyle
```

Comment:
Why is there no style when there are less than 3?

Reply:
I try to add a certain height percentage to every participants section in order for all of them to fit it.

Reply:
Can you pl leave some comment then? The relationship between the 2 things is not very clear.